lib/active_admin/views/components/columns.rb

```diff
@@ -93,19 +93,24 @@ def calculate_columns!
         all_margins_width = margin_size * (span_count - 1)
         column_width = (100.00 - all_margins_width) / span_count
 
-        children.each_with_index do |col, i|
+        columns.each_with_index do |column, i|
           is_last_column = i == (columns_count - 1)
-          col.set_column_styles(column_width, margin_size, is_last_column)
+          column.set_column_styles(column_width, margin_size, is_last_column)
```

Comment:
I'm not sure that this code does the same as the old one. What `children` can be there that are not a `Column`?